backend/src/main/java/ai/giskard/service/InitService.java

```diff
@@ -269,8 +270,8 @@ public void initUsers() {
             }
         });
 
-        if (!featureFlagService.hasFlag(FeatureFlagService.FeatureFlag.AUTH)) {
-            // Given the loop above, we can safely assume that the user at least exists.
+        if (!licenseService.getCurrentLicense().hasFeature(FeatureFlag.AUTH)) {
```

Comment:
I feel that we'll be using this construction a lot, so perhaps it's worth just adding a `hasFeature` method to the `LicenseService` directly